know/rdf/parser/summarizer.py

```diff
@@ -0,0 +1,31 @@
+from .wikidata_parser import WikidataParser
+from .viaf_parser import ViafParser
+class Summarizer:
```

Comment:
Honestly I don't really like this name, but I can't think of anything better so good enough I guess